src/server_manager/messages/master_messages.json

```diff
@@ -1244,8 +1244,13 @@
     }
   },
   "share_invite": {
-    "message": "You\u2019re invited to connect to my Outline server. Use it to access the open internet, no matter where you are. Follow the instructions on your invitation link below to download the Outline App and get connected.",
-    "description": "This string appears as a paragraph within a dialog that provides an invitation to share access to a server. Belongs to the instructions for the user with whom access is being shared."
+    "message": "You're invited to Outline! Follow these instructions to use my server to access the open internet on any device, no matter where you are. No personal information will be collected.&#10;1) Download and install the Outline app for your device:&#10;&#10;  - Android: https://play.google.com/store/apps/details?id=org.outline.android.client&#10;  - iOS: https://itunes.apple.com/app/outline-app/id1356177741&#10;  - Windows: https://s3.amazonaws.com/outline-releases/client/Outline-Client.exe - once downloaded, double click to launch.&#10;  - macOS: https://itunes.apple.com/app/outline-app/id1356178125&#10;  - Linux: https://s3.amazonaws.com/outline-releases/client/Outline-Client.AppImage - once downloaded, make it executable, then double click to launch.&#10;&#10;2) Copy the following access key.&#10;&#10;  $ACCESS_KEY$&#10;&#10;3) In the Outline app, tap the \"+\" sign on the top bar and paste the access key.&#10;&#10;4) Press the \"ADD SERVER\" button to confirm.&#10;&#10;5) A card will be added to your app. Tap \"Connect\" to get connected with the new service you added.",
+    "description": "This string appears as an email invitation instructing new Outline users how to set up and connect to Outline.",
+    "placeholders": {
+      "ACCESS_KEY": {
+        "content": "{accessKey}"
```

Comment:
Can you insert the access key as a link? It may work in some cases, or prevent auto-linking from doing the wrong thing.